<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDeviceDialog</class>
 <widget class="QDialog" name="AddDeviceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QListWidget" name="lstDevices"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLTopic">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lblTopic">
       <property name="text">
        <string>Topic: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTopic">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbType">
       <property name="toolTip">
        <string>Тип устройства</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Переключаемое</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Информационное</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Редактируемое</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QtMaterialCheckBox" name="chkNeedrecognize">
     <property name="text">
      <string>Распознавать</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Сфинкс</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAction">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbObject">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbPretext">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>в</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>на</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbWhere">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>зале</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>кухне</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>туалете</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QtMaterialCheckBox" name="chkNeedResponse">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Отвечать</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="txtResponse">
       <property name="text">
        <string>Ответ: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leResponse">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialCheckBox</class>
   <extends>QCheckBox</extends>
   <header>qtmaterialcheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
